cmd/test.go

```diff
@@ -81,9 +82,14 @@ ibc-test-framework test -s osmosis:v7.0.4 -d juno:v2.3.0 -r rly RelayPacketTest
 		var testCases []func(testName string, cf ibc.ChainFactory, relayerImplementation ibc.RelayerImplementation) error
 
 		for _, testCaseString := range strings.Split(testCasesString, ",") {
-			testCase, err := ibc.GetTestCase(testCaseString)
+			// Prefer the new way of getting legacy test cases,
+			// but fall back to the old way for now.
+			testCase, err := relayertest.GetLegacyTestCase(testCaseString)
```

Comment:
I think we should go one way or the other with this (GetLegacyTestCase or GetTestCase). I prefer the reflection so that the mapping does not need to be maintained. But maybe neither are necessary if we ditch the CLI and require the matrix files instead.